src/ui/properties/TroikaTextNodeEditor.js

```diff
@@ -0,0 +1,361 @@
+// This is a port of https://github.com/protectwise/troika/tree/master/packages/troika-three-text
+// that has been made functional and restyled for Mozilla Hubs and Spoke (hubs.mozilla.com)
+// by @jamesckane at Paradowski Creative (paradowski.com)
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import NodeEditor from "../../../../src/ui/properties/NodeEditor";
+import InputGroup from "../../../../src/ui/inputs/InputGroup";
+import StringInput from "../../../../src/ui/inputs/StringInput";
+import { AlignCenter } from "styled-icons/fa-solid/AlignCenter";
+import SelectInput from "@src/ui/inputs/SelectInput";
```

Comment:
Maybe there is some webpack module that you are using for `@` that's not been commited?